tests/page-objects/default/analytics/analytics.wdio.page.js

```diff
@@ -20,6 +20,12 @@ const targetNumberPercentCount = (targetElement) => targetElement.$('.body .targ
 
 const targetGoalValue = (targetElement) => targetElement.$('.body .count .goal p');
 
+const emptySelection = '.content-pane .item-content.empty-selection';
+
+const emptySelectionError = () => $(`${emptySelection}.selection-error`);
+
+const emptySelectionNoError = () => $(`${emptySelection}:not(.selection-error)`);
```

Comment:
Nice! Just a small one, whenever we have constant variables, then the name is all upper case like this: 
```suggestion
const EMPTY_SELECTION = '.content-pane .item-content.empty-selection';

const emptySelectionError = () => $(`${EMPTY_SELECTION}.selection-error`);

const emptySelectionNoError = () => $(`${EMPTY_SELECTION}:not(.selection-error)`);
```